web/src/views/SubmissionPortal/Components/LoginPrompt.vue

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+
+export default defineComponent({
+  props: {},
+  setup() {
+    return {
+      loginHref: '/login',
+    };
+  },
+});
+</script>
+
+<template>
+  <v-alert
+    class="mb-0"
+    text="text"
+  >
+    <v-container>
+      <v-row align="center">
+        <v-col class="justify-content-center">
+          <p class="body-1">
+            Login with your ORCiD
+            to access the Submission Portal.
+          </p>
+          <p class="body-2 mb-0 text--disabled">
+            Don't have an ORCiD? Individuals can
+            <a
+              href="https://support.orcid.org/hc/en-us/articles/360006973953"
+              target="_blank"
```

Comment:
add rel = noreferrer nopener for extra security.

Reply:
Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.